projects/swimlane/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -541,9 +471,7 @@ export class DatatableComponent implements OnInit, DoCheck, AfterViewInit {
    */
   @HostBinding('class.fixed-row')
   get isFixedRow(): boolean {
-    const rowHeight: number | string = this.rowHeight;
-    return (typeof rowHeight === 'string') ?
-      (<string>rowHeight) !== 'auto' : true;
+    return this.rowHeight !== 'auto';
```

Comment:
a thought, if the function they pass to rowHeight is impure, the row could potentially not be fixed and this check wont catch it

Reply:
The value would still be provided from outside though. As far as the datatable is concerned, it doesn't have to calculate the row height itself.

Reply:
makese sense, gotcha